### PR TITLE
Docs 1829 add missing props hn

### PR DESCRIPTION
**Pull request by hannanelson**
I didn't edit the rest of this topic - just added the missing properties.

### CODE REVIEW

File: modules/ROOT/pages/alerts-on-runtime-manager.adoc [hunk 1]
```diff
@@ -243,15 +243,20 @@ The following properties are available and can be used in the alert Email to, Su
...
-|`${currentValue}` |Current value of metric that triggers the alert |Conditional
+|`${operator}` | An operator, such as: < > = |Conditional
+|`${priority}` |Priority of the notification triggering the alert. |Custom Application Notification
+|`${resource}` | Name of the resource triggering the alert, such as server or app. | All
+|`${severity}` | Severity of the alert. |All
+|`${state}` | State of the resource triggering the alert, such as Running or Disconnected. | All
 |`${threshold}` |Count over timeframe which triggers the alert |Conditional
-|`${message}` |The message from the notification triggering the alert. |Custom Application Notification
-|`${priority}` |The priority of the notification triggering the alert. |Custom Application Notification
+|`${value} | A value associated with an element | Conditional
```
Comment (valkyrie69) [inline modules/ROOT/pages/alerts-on-runtime-manager.adoc]: Need the closing tick mark on ${value}

File: modules/ROOT/pages/alerts-on-runtime-manager.adoc [hunk 2]
```diff
@@ -169,15 +169,15 @@ The available alert events vary depending on the *Server Type* field. The alerts
...
+To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert.
 
 
 
 * *Memory Usage* - Total memory usage in MB on the server.
 +
 image::memory-usage.png[memo]
 +
-To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert. 
+To enable this condition, specify values in the duration and capacity threshold parameters that, when crossed, trigger the alert.
```
Comment (valkyrie69) [inline modules/ROOT/pages/alerts-on-runtime-manager.adoc:178]: This is the same text that's on line 172. There is probably a reason for it. Just checking.

File: modules/ROOT/pages/alerts-on-runtime-manager.adoc [hunk 3]
```diff
@@ -300,11 +305,11 @@ From the *Alerts* page, click an alert name to open its corresponding panel. Fro
 image::alerts-history.png[history]
 
 
-You can also see the alert history record of an individual server or of an application that's deployed to one. 
+You can also see the alert history record of an individual server or of an application that's deployed to one.
 
-. On the *Applications* tab, click the xref:managing-deployed-applications.adoc#applications-tab[*Manage Application* button] (for applications deployed on premises), or click the xref:managing-servers.adoc#settings-page[*Manage Server* button] on the *Servers* tab. 
+. On the *Applications* tab, click the xref:managing-deployed-applications.adoc#applications-tab[*Manage Application* button] (for applications deployed on premises), or click the xref:managing-servers.adoc#settings-page[*Manage Server* button] on the *Servers* tab.
```
Comment (valkyrie69) [inline modules/ROOT/pages/alerts-on-runtime-manager.adoc]: Probably don't need these UI buttons to be links.

**Review by valkyrie69 — APPROVED** [review]
Very minor comments

File: modules/ROOT/pages/alerts-on-runtime-manager.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/alerts-on-runtime-manager.adoc:178]: Seems applicable to both.

**Pull request merged**